frontend/app/package.json

```diff
@@ -14,7 +14,7 @@
     "cy:run": "unset NODE_OPTIONS && cypress run",
     "cy:run-flaky": "yarn cy:run --config integrationFolder=../e2e_flaky/specs",
     "typecheck": "tsc --noEmit",
-    "lint": "eslint --ext .js --ext .jsx --ext .ts --ext .tsx --max-warnings 0 --cache --fix src"
```

Comment:
is the `--fix` arg intentionally removed?

Reply:
Hm that's how it is in develop so I removed. I personally like the --fix when I run it locally and accidentally added it in a separate PR seemingly. I am happy to keep it because I use it but not sure if others use it.

Reply:
Makes sense - let's keep parity with develop